_bibliography/external/Lossy_checkpoint_restart.bib

```diff
@@ -0,0 +1,105 @@
+---
+---
+%% NOTE
+%% The two lines above are required for Jekyll and the jekyll-scholar plugin to correctly parse
+%% and render the references on the web page.
+%% You may remove them when using this file with BibTeX natively in your publications to avoid nasty
+%% BibTeX errors.
+
+%
+% The entries in this file are sorted by YEAR (ascending).
+% If inserting new entries, please put them accordingly.
+%
+%
+% %% Convention for BibTeX keys
+%
+% Construct the BibTeX key according to the following schemes.
+% In the following, please replace these patterns accordingly:
+%
+%   YYYY           year of publication
+%   Aaaa           first author's surname in CamelCase
+%   Bbbb           second author's surname in CamelCase
+%
+% - for single-author publications:   AaaaYYYY
+% - for two-authors publications:     AaaaBbbbYYYY
+% - for three or more authors:        AaaaEtAlYYYY
+%
+% In case an author has published more than one publication in the same year, append lower case
+% letters for the second and ongoing publications:
+%
+%   first publication:   AaaaEtAlYYYY
+%   second publication:  AaaaEtAlYYYYb
+%
+%
+% %% Convention for BibTeX fields
+%
+% Please, stick to the following formatting convention to easy maintainability:
+%
+% - indent BibTeX fields of entries with 2 spaces (NOT tabs!)
+%
+% - order the BibTeX fields in alphabetic order
+%
+% - enclose all values within curly brakets '{...}'
+%
+% - do line breaks at 100 chars; this is ................................................. here -->|
+%
+% - and add a continuation indent of 2 after line breaks
+%
+% - put a colon ',' as well after the last field/value pair of an entry
+%
+%
+% %% Important
+%
+% Please, treat this file not different from usual source code files.
+%
+
+@inproceedings{Ibtesham:2011:VCC:2238436.2238474,
```

Comment:
Thanks,
How to include URL in the text of the report?
Franck

From: Torbjörn Klatt
Reply-To: "JLESC/jlesc.github.io"
Date: Monday, April 11, 2016 at 14:44
To: "JLESC/jlesc.github.io"
Cc: Franck
Subject: Re: [JLESC/jlesc.github.io] Project description: Lossy checkpoint restart (#189)


In _bibliography/external/Lossy_checkpoint_restart.bib<https://github.com/JLESC/jlesc.github.io/pull/189#discussion_r59268098>:

> +%
> +% - enclose all values within curly brakets '{...}'
> +%
> +% - do line breaks at 100 chars; this is ................................................. here -->|
> +%
> +% - and add a continuation indent of 2 after line breaks
> +%
> +% - put a colon ',' as well after the last field/value pair of an entry
> +%
> +%
> +% %% Important
> +%
> +% Please, treat this file not different from usual source code files.
> +%
> +
> +@inproceedings{Ibtesham:2011:VCC:2238436.2238474,


Do you mind adjusting the BibTeX keys to the pattern described in the commend above?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/JLESC/jlesc.github.io/pull/189/files/c963ddde8e7391bc6156f3f34c2cc01b27905bf9#r59268098>